funcr/example_formatter_test.go

```diff
@@ -41,17 +41,17 @@ type stdoutlogger struct {
 }
 
 func (l stdoutlogger) WithName(name string) logr.LogSink {
-	l.Formatter.AddName(name)
+	l.AddName(name)
```

Comment:
Is this something that we might want to enable as hint in Kubernetes? Or is it too subjective?